.github/scripts/check-commit-titles.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
+# shellcheck disable=SC2317
```

Comment:
Out of curiosity, which part is wrongly marked as unreachable?

Reply:
All the code in check_structure, check_forbidden_chars, check_fixup. ShellCheck has trouble due to the functions being called by variable name.

Reply:
Ah right, makes sense!